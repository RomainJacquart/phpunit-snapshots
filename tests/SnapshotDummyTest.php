<?php

namespace Madewithlove\PhpunitSnapshots;

class SnapshotDummyTest extends TestCase
{
    public function testCanUseSnapshotTesting()
    {
        $this->assertEqualsSnapshot(['foo', 'bar']);
    }
}
