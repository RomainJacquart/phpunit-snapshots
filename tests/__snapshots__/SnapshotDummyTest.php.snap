{
    "testCanUseSnapshotTesting-0": [
        "foo",
        "bar"
    ]
}